SU-SWS/stanford-jumpstart-deployer
# Adds patch to fix bug where when users add css_injector files via con…

…text and aggregation is enabled.

# Status
- Ready for review

# Needed By (Date)
- Please merge before any new releases to products
- March 15?

# Urgency
- Not super urgent but it is possible to reproduce in the sites environment.

# Steps to Test

1. Build any product without this patch
2. Create a css injector file. I like to add the style `* { background: #F0F; }`
3. Create a context and add a reaction of css_injector to place the css_injector you created
4. Turn on aggregation
5. Clear caches
6. View any page in the front end that matches your new context and you SHOULD NOT see a difference in backgrounds.
7. Disable aggregation on css files
8. Clear caches
9. View any page in the front end that matches your new context and you SHOULD see a difference in backgrounds.
10. Apply patch
11. Rinse and repeat with css always applied.

# Affected Projects or Products
 - Everything from sites stack to JSE

# Associated Issues
- Was first discovered on Iranian Studies.
